### ISSUE TITLE
typescript-pundle missing lib/index.js

### ISSUE THREAD

**Issue by appsforartists** [steelbrain/pundle#83]
`typescript-pundle` is missing its exports!

```
> yarn add typescript-pundle
> ls node_modules/typescript-pundle

README.md
node_modules
package.json
spec
```

Maybe you forgot to run bootstrap before publishing?

**Comment by steelbrain** [comment]
What the... lol

Fix incoming soon

**Comment by steelbrain** [comment]
Fixed in `typescript-pundle@1.0.1`
